src/commands/kv/bucket/mod.rs

```diff
@@ -21,23 +21,23 @@ use twox_hash::XxHash64;
 
 use cloudflare::endpoints::workerskv::write_bulk::KeyValuePair;
 
-use crate::settings::toml::Target;
+use crate::settings::toml::Site;
 
 pub const KEY_MAX_SIZE: usize = 512;
 // Oddly enough, metadata.len() returns a u64, not usize.
 pub const VALUE_MAX_SIZE: u64 = 10 * 1024 * 1024;
 
 // Returns the hashed key and value pair for all files in a directory.
 pub fn directory_keys_values(
-    target: &Target,
+    site: Option<Site>,
```

Comment:
running down this, we use this to find out whether to include or exclude files. i'd be inclined to pass those two parameters instead of an `Option<Site>`

Reply:
if i try to do this, then `kv::bucket::sync` has an unwieldy number of parameters. going to leave this as is for another refactor

Reply:
it can be destructured in kv::bucket::sync before being passed  to this fn.